clicommand/agent_start.go

```diff
@@ -327,6 +335,13 @@ var AgentStartCommand = cli.Command{
 			cfg.NoPlugins = true
 		}
 
+		// Initially, plugin validation must be enabled via an experiment
```

Comment:
Does it need to be an experiment? Can it just be a config option with a `false` default value?